### PR TITLE
removed not used console.logs

### PR DESCRIPTION
**Pull request by SimonEntholzer**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context

In the code some `console.logs()` were left, which to some part interfere with the client test logs, as they print a lot of unuseful information. It's also not best practice to still see console.logs in production.

### Description

This PR removes unnecessary `console.logs()` from the client code.

### CODE REVIEW

File: src/main/webapp/app/course/course-scores/course-scores.component.ts [hunk 1]
```diff
@@ -236,13 +236,9 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
                 }
             }
         }
-        console.log(`Performed ${noOfComparisons} comparisons between old and new calculation method.`);
-        console.log(`Found ${noOfPointDifferencesFound} point differences between old and new calculation method.`);
-        console.log(`Found ${noOfScoreDifferencesFound} score differences between old and new calculation method.`);
```
Comment (FrankeLukas) [inline src/main/webapp/app/course/course-scores/course-scores.component.ts:241]: The `noOfPointDifferencesFound` variable is now unused.

File: src/main/webapp/app/exam/exam-scores/exam-scores.component.ts [hunk 2]
```diff
@@ -704,13 +704,9 @@ export class ExamScoresComponent implements OnInit, OnDestroy {
                 }
             }
         }
-        console.log(`Performed ${noOfComparisons} comparisons between old and new calculation method.`);
-        console.log(`Found ${noOfPointDifferencesFound} point differences between old and new calculation method.`);
-        console.log(`Found ${noOfScoreDifferencesFound} point differences between old and new calculation method.`);
```
Comment (FrankeLukas) [inline src/main/webapp/app/exam/exam-scores/exam-scores.component.ts:709]: Same here.

**Review by julian-christl — APPROVED** [review]
Apart from @FrankeLukas notes, looks good.
Please check in general the "Files changed" section to see all "unused variables" hints.

Didn't test this due to the scope of this PR.

**Review by ndricimrr — COMMENTED** [review]
Except for the warnings mentioned above, looks good 👍

File: src/main/webapp/app/course/course-scores/course-scores.component.ts (hunk 1 above)
Comment (SimonEntholzer) [reply src/main/webapp/app/course/course-scores/course-scores.component.ts:241]: good catch, I removed those variables

**Review by julian-christl — APPROVED** [review]

**Review by jhainzlschmid — APPROVED** [review]
Only console.logs have been removed (and their corresponding variables). Looks good to me

**Review by iozbeyli — APPROVED** [review]

**Review by FrankeLukas — APPROVED** [review]
Only console logs removed, so this shouldn't need manual testing.

**Review by jpbernius — APPROVED** [review]

**Pull request merged**